packages/eas-cli/src/submissions/ios/AppProduce.ts

```diff
@@ -62,9 +64,86 @@ Learn more here: https://expo.fyi/bundle-identifier`
     language: sanitizeLanguage(language) ?? 'English',
   };
 
+  if (USE_APPLE_UTILS) {
+    return await runProduceExperimentalAsync(options);
+  }
+
   return await runProduceAsync(options);
 }
 
+async function isProvisioningAvailableAsync(requestCtx: RequestContext): Promise<boolean> {
+  const session = await Session.getAnySessionInfo();
+  const username = session?.user.emailAddress;
+  const [user] = await User.getAsync(requestCtx, { query: { filter: { username } } });
```

Comment:
I worry about this since the User object has username and email, but you cannot query against `email`, and I haven't seen a case where username and email are different. Maybe make a TODO about investigating this further in the future.